DependencyInjection/HttplugExtension.php

```diff
@@ -54,6 +66,10 @@ protected function configureClients(ContainerBuilder $container, array $config)
                 $first = $name;
             }
 
+            if (isset($config['_inject_collector_plugin'])) {
+                array_unshift($arguments['plugins'], 'httplug.collector.history_plugin');
```

Comment:
true. hm. it would be cool to see whether requests where served from
cache or the actual backend. we could try to have two history plugins
and see whether a request goes to the client or not, and also see
whether there is a redirect to serve one answer.